### ISSUE TITLE
Can't post at the main BOINC forum

### ISSUE THREAD

**Issue by computezrmle** [BOINC/boinc#4635]
When I try to send a post to the forum
https://boinc.berkeley.edu/forum_index.php
The forum replies with a message that my email address needs to be verified.
When I click the link to initiate the verification I get a page claiming a verification email has been sent to my address.
But I never get that email.

- Forum login succeeds
- email address given there is correct
- email postbox receives mail (from different senders)
- spam folder contains no email from BOINC

Can't even make the forum admins aware since this would require to post a message.

**Comment by Ageless93** [comment]
David, for when you check this and need his account, it's as far as I can see https://boinc.berkeley.edu/show_user.php?userid=960148

**Comment by davidpanderson** [comment]
I manually set that user's "email validated" flag.  However, I don't know why he didn't receive the email.  It works for me.  Maybe if other people could try this a pattern would emerge.

**Comment by computezrmle** [comment]
Thanks.
Was able to post a comment now.

BTW: I receive email from github.com/BOINC/boinc via the same email address and this works fine.

**Comment by computezrmle** [comment]
Sorry to reopen this again.

For a couple of hours I can't login to the main BOINC forum any more.
https://boinc.berkeley.edu/forum_index.php

- Login was OK this morning
- nothing has been changed on my side (was out for some time)
- requested "Instructions for resetting your password ..." but that email didn't arrive.
- checked the SPAM folder -> no email from BOINC
- checked if other senders can send emails to that address -> they can
- can't post the problem at the BOINC forum since I can't log in
- BOINC projects where I'm attached to work fine

Please forward the issue to a BOINC forum admin.

**Comment by AenBleidd** [comment]
@davidpanderson, could you please check this?
I suppose this is the correct account link: https://boinc.berkeley.edu/show_user.php?userid=960148

**Comment by davidpanderson** [comment]
I sent him a new password.

Apparently there's still a problem with emails sent from boinc.berkeley.edu.
It works to some domains and not others.
I tried this: https://mxtoolbox.com/blacklists.aspx?AG=GBL
and it didn't show boinc.berkeley.edu as being on any blacklists.

**Comment by computezrmle** [comment]
Thanks, could login with the intermediate PW, change it to a new one and log in with that new one.
This solves the primary issue.

I got an email from @davidpanderson from the domain berkeley.edu.
So this domain is not on the blacklist of my ISP.
If it would it should have been gone to the SPAM folder.

I did some more investigation and found a PW for seti@home in my firefox PW store. Seti is a subdomain of berkeley.edu but I didn't use the email as username there.
Nonetheless, could there be an unwanted side effect?
